src/sagas/projects.js

```diff
@@ -87,6 +90,27 @@ export function* updateProjectSource() {
   yield* saveCurrentProject();
 }
 
+export function* loadAndBeautifyProjectSource() {
+  const state = yield select();
+  const currentProject = getCurrentProject(state);
+  const allFormattedSources = yield all(
+    Reflect.ownKeys(currentProject.sources).map((language) => {
+      const source = currentProject.sources[language];
+      return call(
+        beautifySource,
+        source,
+        language,
+      );
+    }),
+  );
+  const sourcesMap = new Map();
```

Comment:
I think there’s a good argument that a `Map` is a more appropriate data structure for this use case than a plain object, but given that we use plain objects in action payloads to communicate the same overall data structure elsewhere in the code, I think it’s best to be consistent here. I think the code in the reducer will be fine with it either way. Thoughts?

Reply:
yea i remember trying an object and it not working for me, but seems to work now 🤷‍♀️